src/FormType/AbstractFormType.php

```diff
@@ -71,4 +112,52 @@ public function onLoadFormField(LoadFormFieldEvent $event): void
     public function onGetForm(GetFormEvent $event): void
     {
     }
+
+    protected function onUpdate(ProcessFormDataEvent $event): void
+    {
+        $formContext = $this->getFormContext();
+        $oldData = $formContext->getData();
+        $newData = $event->getSubmittedData();
+        $validKeys = array_keys($oldData);
+
+        $setData = [];
+
+        foreach ($newData as $key => $newValue)
```

Comment:
Schau dir hierzu nochmal den storeValues-Part in der Form-Klasse an (so Zeile 480ff), gerade die Datums-Sache ist eventuell noch spannend. Sonst sehr schick!